src/global/tracking/tracking.cc

```diff
@@ -166,9 +186,27 @@ void InitPlugin(JApplication* app) {
                                                               },
                                                               app));
 
+  app->Add(new JOmniFactoryGeneratorT<ActsTrajectoriesMerger_factory>(
+      "CentralB0CKFActsTrajectories",
+      {
+          "CentralCKFActsTrajectories",
+          "B0TrackerCKFActsTrajectories",
+      },
+      {
+          "CentralAndB0TrackerCKFActsTrajectories",
+      },
+      app));
+
   app->Add(new JOmniFactoryGeneratorT<IterativeVertexFinder_factory>(
-      "CentralTrackVertices", {"CentralCKFActsTrajectories", "ReconstructedChargedParticles"},
-      {"CentralTrackVertices"}, {}, app));
+      "CentralTrackVertices",
+      {
+          "CentralAndB0TrackerCKFActsTrajectories",
```

Comment:
same here, can you move this block down, and rename the output from "CentralTrackVertices" to "CentralAndB0TrackVertices"?

Reply:
The point here is that you can't constrain the vertices well without the B0.

Reply:
(for posterity) The current state of `CentralTrackVertices` is 100% agreement with the original reference in epic-capybara.